autogpt_platform/backend/backend/notifications/notifications.py

```diff
@@ -139,6 +162,22 @@ def _parse_message(self, message: str) -> NotificationEvent | None:
             logger.error(f"Error parsing message due to non matching schema {e}")
             return None
 
+    def _process_admin_message(self, message: str) -> bool:
+        """Process a single notification, sending to an admin, returning whether to put into the failed queue"""
+        try:
+            parsed = self._parse_message(message)
+            if not parsed:
+                return False
+            event = parsed.event
+            model = parsed.model
+            recipient_email = settings.config.refund_notification_email
+            self.email_sender.send_templated(event.type, recipient_email, model)
+            logger.info(f"Processing notification: {model}")
```

Comment:
model= ?

Reply:
model is like the data you passed in, so like who you're refunding etc